### ISSUE TITLE
What is the training set?

### ISSUE THREAD

**Issue by Bowen1911** [hemingkx/TokenSkip#8]
In the sample code, the training set is GSM8K, and the test set is GSM8K and MATH-500. Among them, is GSM8K at risk of data leakage? Or is the train set using other data, and I found no clues in the paper. If I am missing something, please correct me, thx!
